hummingbot/client/hummingbot_application.py

```diff
@@ -106,22 +110,41 @@ def __init__(self, client_config_map: Optional[ClientConfigAdapter] = None):
         self._pmm_script_iterator = None
         self._binance_connector = None
         self._shared_client = None
+        self._mqtt: MQTTGateway = None
 
         # gateway variables and monitor
         self._gateway_monitor = GatewayStatusMonitor(self)
 
-        command_tabs = self.init_command_tabs()
-        self.parser: ThrowingArgumentParser = load_parser(self, command_tabs)
-        self.app = HummingbotCLI(
-            self.client_config_map,
-            input_handler=self._handle_command,
-            bindings=load_key_bindings(self),
-            completer=load_completer(self),
-            command_tabs=command_tabs
-        )
+        if self.client_config_map.mqtt_bridge.mqtt_autostart:
+            try:
+                self._mqtt = MQTTGateway(self)
+                self._mqtt.start_notifier()
+                self._mqtt.start_commands()
+                self._mqtt.start_event_fw()
+                self._mqtt.patch_logger_class()
+                self._mqtt.run()
+            except Exception as e:
+                self.logger().error(e)
+
+        if os.getenv('HBOT_HAS_TUI') in ('1', 'Yes', 'yes', 'YES', 'Y', '', None):
+            command_tabs = self.init_command_tabs()
+            self.parser: ThrowingArgumentParser = load_parser(self, command_tabs)
```

Comment:
I think we'll have to define a dummy parser here as the calls to `self.parser` don't check for existence that I can see